### ISSUE TITLE
Other types of cobble stone can't be used as cobblestone

### ISSUE THREAD

**Issue by Guineapigboyx** [uthw/minecraft-modpack#55]
Like you can't use garbo cobblestone for a furnace. _there are more this is just what came to mind_

This is mainly an issue with terraforged worlds since they normally give out custom types of stone.

**Comment by uthw** [comment]
Agreed. They also look very same-y. I could mess with backporting the new Create textures where this isn't the case.

I'll also add the Create Deco mod which hopefully has brick variants and more for these materials. In our server I gathered like 3 stacks of weathered limestone for building before realizing I can't really do anything with it.
